tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php

```diff
@@ -530,4 +671,40 @@ public function getBigQueryClient(array $externalCredentials): BigQueryClient
         ]);
         return $bqClient;
     }
+
+    private function createExternalTable(string $description): void
+    {
+        $gcsClient = new StorageClient([
+            'keyFile' => $this->getCredentialsArray(),
+        ]);
+
+        $filePath = __DIR__ . '/../../_data/languages.csv';
+        $retBucket = $gcsClient->bucket('my-external-table-bucket');
+        $retBucket->exists();
+
+        $file = fopen($filePath, 'rb');
+        if (!$file) {
+            throw new ClientException("Cannot open file {$file}");
+        }
+        $object = $retBucket->upload(
+            $file,
+            [
+                'name' => 'languages.csv',
+            ]
+        );
+
+        // this must be done in a real situation by a user who registers an external bucket
+        $object->acl()->add('user-'.BQ_DESTINATION_PROJECT_SERVICE_ACC_EMAIL, 'READER');
```

Comment:
Tu je blbe, ze ked si vytvorim ex table z filu tak este musi mat ten cielovy projket pravo na ten file, nestaci ze ho linknem do cieloveho projektu. Spomeniem to este v navode. Su teda dve moznosti co by mal navod obsahovat a to bud teda dam grat na view celemu bucketu, a potom sa uz viac nemusim starat alebo to dam len jednemu suboru o ktorom viem ze budem robit z neho ext tabulku